src/components/image/Image.constants.ts

```diff
@@ -1,11 +1,9 @@
 import { TShapeBorderRadius } from '../../foundations/shape';
 
-import TImageSize from './Image.types';
-
-const IMAGE_SIZES: TImageSize[] = ['auto', 'full'];
+const DEFAULT_IMAGE_BORDER_RADIUS: TShapeBorderRadius = 4;
 
-const DEFAULT_IMAGE_SIZE: TImageSize = 'auto';
+const DEFAULT_IMAGE_WIDTH = 'auto';
```

Comment:
why remove the types from here?

Reply:
this is since new typing for the `width` and `height` are just `number | string`